lib/pf/web.pm

```diff
@@ -201,10 +201,11 @@ sub generate_scan_start_page {
     my $logger = get_logger();
 
     $portalSession->stash({
-        timer           => $Config{'scan'}{'duration'},
+        # Hardcoded here since the scan section is gone.
+        # In case this codepath is still called (pf::web::release is still called by scan in some cases)
+        timer           => 60,
```

Comment:
Not even sure that codepath is still being used so I don't think we should really put efforts in it.

Would like to remove it completely but I'm not confident enough about it